src/indexes/tag.cc

```diff
@@ -167,16 +167,14 @@ absl::StatusOr<bool> Tag::RemoveRecord(const InternedStringPtr& key,
 }
 
 int Tag::RespondWithInfo(ValkeyModuleCtx* ctx) const {
-  auto num_replies = 6;
+  auto num_replies = 8;
```

Comment:
These source code changes were merged a while ago. I'd merge them in just to make the diff more clear